.vscode/settings.json

```diff
@@ -43,6 +43,6 @@
   "typescript.surveys.enabled": false,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
What does this do?

Reply:
this is coming from VSCode latest version https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto. It's the same as `true`, it should be safe to commit it

<img width="759" alt="Screenshot 2023-12-13 at 10 09 02" src="https://github.com/artsy/metaphysics/assets/11945712/e4575635-5f06-44c6-a932-ea362b0995f0">

Reply:
Good catch! I noticed and un-committed this a couple of times but it still snuck through 🙈  

I hadn't looked into the cause yet, thanks @MounirDhahri. Sounds like it's okay to actually leave this in.